content/docs/tasks/security/health-check/index.md

```diff
@@ -41,13 +43,16 @@ See the [authentication policy task](/docs/tasks/security/authn-policy/) for det
 
 ## Deploying Citadel with health checking
 
-To enable health checking, redeploy Citadel using the configuration in `istio-citadel-with-health-check.yaml`:
+To enable health checking, redeploy Citadel with helm:
 
 {{< text bash >}}
-$ kubectl apply -f install/kubernetes/istio-citadel-with-health-check.yaml
+$ helm template install/kubernetes/helm/istio --name istio --namespace istio-system \
+-x charts/security/templates/deployment.yaml \
+--set global.mtls.enabled=true --set security.citadelHealthCheck=true > citadel-health-check.yaml
+$ kubectl apply -f citadel-health-check.yaml
 {{< /text >}}
 
-## Verifying the health checker is working
+## Verifying the health checking is working
```

Comment:
Verifying that health checking is working

Reply:
```suggestion
## Verify that health checking works
```